src/AWS.Deploy.CLI/Commands/GenerateDeploymentProjectCommand.cs

```diff
@@ -98,9 +100,11 @@ public async Task ExecuteAsync(string saveCdkDirectoryPath, string projectDispla
             await _cdkProjectHandler.CreateCdkProjectForDeployment(selectedRecommendation, _session, saveCdkDirectoryPath);
             await GenerateDeploymentRecipeSnapShot(selectedRecommendation, saveCdkDirectoryPath, projectDisplayName);
 
-            var saveCdkDirectoryInfo = _directoryManager.GetDirectoryInfo(saveCdkDirectoryPath);
+            var saveCdkDirectoryFullPath = _directoryManager.GetDirectoryInfo(saveCdkDirectoryPath).FullName;
```

Comment:
this seems not required, you can keep the DirectoryInfo object and pass the required property.

Reply:
I only need the full directory path and not the full `DirectoryInfo` object. Hence only capturing the necessary information.